### PR TITLE
Updates README instructions for submodule/common use.

### PR DESCRIPTION
**Pull request by iambmelt**
This change updates the `README` with instructions for including ADAL in your project.

The include-as-zip option has been removed because it's incredibly _easy_ to **mess up**.
Brief explanation:
- Downloading as zip via GitHub doesn't include the submodule, only an empty directory
- The submodule/branch/SHA being used by ADAL is contingent on the branch being used
- The submodule ref in `.gitmodules` may not necessarily `ref` to `HEAD`, so you'd have to navigate to the proper commit in the version tree before download **OR** `git clone` it (which is what we were trying to prevent having to do in the first place)

`git clone` avoids all of these headaches. Just do that!

### CODE REVIEW

**Comment by iambmelt** [comment]
The `README` still mentions the `hello` sample app. This doesn't exist anymore and the `README` should no longer mention it.

File: README.md [hunk 1]
```diff
@@ -67,18 +67,18 @@ To build with Gradle,
 * Clone this repo in to a directory of your choice
 * Setup emulator with SDK 23
 * Go to the root folder where you cloned this repo
-* To run the sample app, connect the test device and run the command: `./gradlew :sample:installDebug`
-* You should see app 'hello' installed in the test device
-* Enter test user credentials to try
+* To run the sample app, connect the test device and run the command: `./gradlew :userappwithbroker:installDebug`
+* You should see app 'Fancy ADAL Test App' installed on the test device
+* Select an authority, [optionally] enter a login hint and/or query parameters, and clikc `Acquire Token` to enter credentials with AAD
```
Comment (kreedula) [inline README.md]: Typo clikc -> click

Comment (iambmelt) [reply README.md]: Oops. Thanks @kreedula  😄

Comment (iambmelt) [reply README.md]: Fixed via 831a256ed22530a555228ee898f70923b8ab7f0c

**Comment by iambmelt** [comment]
Since these instructions mention the POM, I'll investigate reintroducing it today

**Pull request merged**